openlibrary/i18n/__init__.py

```diff
@@ -30,6 +30,49 @@ def _compile_translation(po, mo):
         raise e
 
 
+def _validate_catalog(catalog):
+    validation_errors = []
+    for message in catalog:
+        if message.fuzzy:
+            if message.lineno:
+                validation_errors.append(
+                    f'  Line {message.lineno}: "{message.string}" is fuzzy.'
```

Comment:
Something like this would be nice cause folks can ctrl-click on it in gitpod/vscode.

```suggestion
f'openlibrary/i18n/te/messages.po:{message.lineno}: "{message.string}" is fuzzy.'
```